src/bilder/images/vault/files/traefik/static_config.yaml

```diff
@@ -0,0 +1,21 @@
+---
+entryPoints:
+  http:
+    address: ":80"                            # Create the HTTP entrypoint on port 80
+    http:
+      redirections:                           # HTTPS redirection (80 to 443)
+        entryPoint:
+          to: "vault"                         # The target element
+          scheme: "https"
+  vault:
+    address: ":443"
+providers:
+  file:
+    filename: /etc/traefik/vault.yaml
```

Comment:
Is this a file that we need to provide as well? Or did this show up with traefik? I presume it looks something like [this](https://doc.traefik.io/traefik-enterprise/providers/vault-kv/#provider-configuration).

Reply:
```
vagrant@bullseye:~$ sudo ls /etc/traefik/*.yaml
/etc/traefik/traefik.yaml  /etc/traefik/vault.yaml
```

Reply:
```
vagrant@bullseye:~$ cat /etc/traefik/vault.yaml 
{
  "http": {
    "routers": {
      "vault-router": {
        "rule": "Host({{ env \"DOMAIN\"}})",
        "service": "vault-service",
        "tls": {
          "cert_resolver": "letsencrypt_resolver"
        }
      },
      "vault-healthcheck": {
        "rule": "PathPrefix(\"/v1/sys/health\")",
        "service": "vault-service",
        "tls": {
          "cert_resolver": "letsencrypt_resolver"
        }
      }
    },
    "services": {
      "vault-service": {
        "load_balancer": {
          "servers": [
            {
              "url": "https://localhost:8200"
            }
          ],
          "servers_transport": "vault-transport"
        }
      }
    }
  }
}
```

Reply:
👍 I follow now.